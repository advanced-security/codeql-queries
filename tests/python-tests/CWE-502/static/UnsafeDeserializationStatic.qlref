CWE-502/UnsafeDeserializationStatic.ql