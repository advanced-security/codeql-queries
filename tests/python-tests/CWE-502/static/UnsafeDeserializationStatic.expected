| unsafe.py:15:18:15:24 | ControlFlowNode for my_data | Usage of Deserializing function |
| unsafe.py:21:22:21:27 | ControlFlowNode for handle | Usage of Deserializing function |
| unsafe.py:24:23:24:28 | ControlFlowNode for handle | Usage of Deserializing function |
| unsafe.py:27:22:27:27 | ControlFlowNode for handle | Usage of Deserializing function |
